layouts/ContestsColumns.tsx

```diff
@@ -0,0 +1,247 @@
+                {column.getIsSorted() === 'desc' && <ChevronDown size={16} />}
+              </div>
+              <p>CTFtime Rating</p>
+            </div>
+          </Button>
+        </div>
+      )
+    },
+    cell: ({ row }) => {
+      // eslint-disable-next-line
+      const [open, setOpen] = useState(false)
+
+      return (
+        <div className="flex justify-between px-4">
+          <ContestSocial
+            open={open}
+            setOpen={setOpen}
+            place={row.original.place}
+            name={row.original.name}
+            ctfPoints={row.original.ctfPoints}
+          />
+          <div className="flex flex-col items-center justify-center">
+            {row.original.writeupTag && (
+              <TooltipProvider>
+                <Tooltip>
+                  <TooltipTrigger asChild>
+                    <Link href={`/tags/${row.original.writeupTag}`}>
+                      <Button variant="ghost" className="h-fit w-fit p-2">
+                        <BookOpen size={16} />
+                      </Button>
+                    </Link>
+                  </TooltipTrigger>
+                  <TooltipContent>View writeups</TooltipContent>
+                </Tooltip>
+              </TooltipProvider>
+            )}
+            <DropdownMenu modal={false}>
+              <DropdownMenuTrigger asChild>
+                <Button variant="ghost" className="h-8 w-8 p-0">
+                  <span className="sr-only">Open menu</span>
+                  <MoreHorizontal className="h-4 w-4" />
+                </Button>
+              </DropdownMenuTrigger>
+              <DropdownMenuContent align="end">
+                <DropdownMenuLabel>Actions</DropdownMenuLabel>
+                <DropdownMenuSeparator />
+                {row.original.ctftimeId && (
+                  <Link
+                    href={`https://ctftime.org/event/${row.original.ctftimeId}`}
+                  >
+                    <DropdownMenuItem className="flex justify-between">
+                      View on CTFtime
+                      <DropdownMenuShortcut>
+                        <Flag size={16} className="ml-4" />
+                      </DropdownMenuShortcut>
+                    </DropdownMenuItem>
+                  </Link>
+                )}
+                <button onClick={() => setOpen(!open)} className="w-full">
+                  <DropdownMenuItem className="flex justify-between">
+                    Share
+                    <DropdownMenuShortcut>
+                      <Share size={16} className="ml-4" />
+                    </DropdownMenuShortcut>
+                  </DropdownMenuItem>
+                </button>
+              </DropdownMenuContent>
+            </DropdownMenu>
+          </div>
+          <div className="flex flex-col items-end justify-center">
+            <span className="flex items-center justify-end gap-1 text-base font-bold text-foreground">
+              {row.original.ctftimeRating !== undefined &&
```

Comment:
There’s no `ctftimeRating === undefined`, should it check for `ctftimeRating && ctftimeRating !== 0` (or just an implicit `ctftimeRating`) instead?